publication-rest/src/main/java/no/unit/nva/publication/update/PublishingRequestResolver.java

```diff
@@ -117,7 +117,6 @@ private TicketEntry persistPublishingRequest(
         throws ApiGatewayException {
         return customerAllowsPublishingMetadataAndFiles()
                    ? publishingRequest
-                         .approveFiles()
```

Comment:
Complete + approveFiles is always done at the same time, moving `approveFiles()` directly to` comlete()` method